portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/structure-mobile-card/structure-mobile-card.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-card [ngClass]="{ 'last-card': isLast }">
```

Comment:
'review-step' class is removed here which corresponds to 'isReviewStep' input variable. If it is not going to be used, maybe we can have it removed from the scss and ts file?